CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to `jiant`
```

Comment:
Please rebase. Many of these changes are irrelevant mismatch junk.

Reply:
> Did you mention having some tool/trick to run indexing/vocab-building only once per task-model pair?

Previously, if a task has task-specific vocabulary, but there are already vocabulary from previous runs in that experiment. The preprocessing will not create vocabulary for this task, despite it's missing in the saved vocabulary, hence causing an error. We add reload_vocab to force the preprocess always create vocabulary anew. 

However, if we initialize each experiment with a dummy run that just load every task. The saved vocabulary will have everything needed in the following runs. So every run will load correctly, and reload_vocab is no longer needed.